packages/core/components/AnnotationPicker/index.tsx

```diff
@@ -4,8 +4,9 @@ import { useSelector } from "react-redux";
 import ListPicker from "../ListPicker";
 import { ListItem } from "../ListPicker/ListRow";
 import { TOP_LEVEL_FILE_ANNOTATION_NAMES } from "../../constants";
-import Annotation from "../../entity/Annotation";
-import { metadata, selection } from "../../state";
+import Annotation, { AnnotationName } from "../../entity/Annotation";
+import { uniqBy } from "lodash";
```

Comment:
This import belongs with the third party imports at the top (alphabetical order)

Reply:
resolved in [0faf718](https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/110/commits/0faf718ed4318494cd1d819ddd75348c06910a2c)